packages/vm/src/context.rs

```diff
@@ -308,7 +326,9 @@ fn create_unmanaged_context_data<S: Storage, Q: Querier>() -> *mut c_void {
         storage: None,
         querier: None,
         #[cfg(feature = "iterator")]
-        iter: None,
+        iter: HashMap::new(),
+        #[cfg(feature = "iterator")]
+        iter_count: 0,
```

Comment:
isn't that always `iter.len()`? In this case this field can be removed

Reply:
Updated to use that rather than explicit counter.